app/components/Article/Article.module.scss

```diff
@@ -0,0 +1,83 @@
+@use '@/styles/mixins.scss' as mix;
+@use '@/styles/variables.scss' as var;
+
+.article {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var.$color-700-red;
+  border-radius: var.$border-radius;
+  height: 50vh;
```

Comment:
It’s better to use dvh. You can take a look at the video  https://www.youtube.com/watch?v=xl9R8aTOW_I